langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/ChatRequest.java

```diff
@@ -100,4 +102,15 @@ public ChatRequest build() {
             return new ChatRequest(this);
         }
     }
+
+    /**
+     * TODO
+     * @param userMessage
+     * @return
+     */
+    public static ChatRequest from(String userMessage) { // TODO needed? keep this method instead of *ChatModel.chat(String, ?)
```

Comment:
Right now it is useful mostly in our ITs where we need to check that exactly `chat(ChatRequest)` method was called with exactly XYZ parameters.
Since we decided to keep `chat(String)`, this method does not make much sense for the library users, so I will move this method out of this class into test helpers.